contrib/hack/update-generated-openapi.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+
+
+SC_PREFIX="github.com/kubernetes-incubator/service-catalog/"
+
+INPUT_DIRS=(
+  # kube apis
+  $(
+    grep --color=never -rl '+k8s:openapi-gen=' vendor/k8s.io/kubernetes | \
+    xargs -n1 dirname | \
+    sed "s,^vendor/,," | \
+    sort -u
+  )
+
+  # service-catalog apis
+  $(
+    grep --color=never -rl '+k8s:openapi-gen=' pkg | \
+    xargs -n1 dirname | \
+    sed "s,^,${SC_PREFIX}," | \
+    sort -u
+  )
+)
+
+INPUT_DIRS=$(IFS=,; echo "${INPUT_DIRS[*]}")
+
+genopenapi \
```

Comment:
Oversight on my part when re-using this script from openshift, will fix: `s,genopenapi,./bin/openapi-gen,`. Will also ensure the binary exists / is built before getting to this point in the script